### PR TITLE
chore(docs): grammatical tweaks to migration guide

### PR DESCRIPTION
**Pull request by thatrobotdev**
- Moved a lot of sentences into active voice
- Removed unnecessary words and sentences
- Fixed some grammatical errors
- Removed potentially confusing emojis as a literary device
- Split up some long sentences into shorter ones
- Added a link to https://www.gatsbyjs.org/blog/2018-06-08-life-after-layouts/ in the layouts section to help people move from Gatsby's layout behavior in v1
- Capitalized instances of "Babel" that were lowercase

### CODE REVIEW

File: docs/docs/migrating-from-v1-to-v2.md [hunk 1]
```diff
@@ -765,13 +761,13 @@ Because the outer wrapper `div` was removed, you can no longer use the `outerWra
...
 
-Similarly, if you have created any CSS styling rules referencing the `gatsby-image-outer-wrapper` class, you should merge those styles into the `gatsby-image-wrapper` class.
+If you have created any CSS styling rules referencing the `gatsby-image-outer-wrapper` class, you should merge those styles into the `gatsby-image-wrapper` class.
 
 ## Resolving Deprecations
 
 ### Import Link from Gatsby
 
-All components and utility functions from `gatsby-link` are now exported from `gatsby` package. Therefore you should import it directly from `gatsby`.
+All components and utility functions from `gatsby-link` are now exported from `gatsby` package. So, you should import it directly from `gatsby`.
```
Comment (m-allanson) [inline docs/docs/migrating-from-v1-to-v2.md]:
```suggestion
All components and utility functions from `gatsby-link` are now exported from the `gatsby` package. So, you should import it directly from `gatsby`.
```

File: docs/docs/migrating-from-v1-to-v2.md [hunk 2]
```diff
@@ -4,17 +4,17 @@ title: Migrating from v1 to v2
 
 Looking for the v1 docs? [Find them here](https://v1.gatsbyjs.org/).
 
-> This document is a work in progress. Have you upgraded your site and run into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
+> This document is a work in progress. Have you upgraded your site and ran into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
```
Comment (m-allanson) [inline docs/docs/migrating-from-v1-to-v2.md]: I don't think _ran_ is right here? Maybe we could shorten the sentence though:

```suggestion
> This document is a work in progress. Have you run into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
```

File: docs/docs/migrating-from-v1-to-v2.md [hunk 3]
```diff
@@ -999,8 +995,7 @@ The signature for using createRemoteFileNode changed in v2, it now expects a new
 
 ### Only allow defined keys on the node internal object
 
-The node internal object isn't meant for adding node data. Those should be added to the top-level object. We
-didn't document this in v1 nor validate against it but are now for v2.
+The node internal object isn't meant for adding node data. Those should be added to the top-level object. We now validate against it for v2.
```
Comment (m-allanson) [inline docs/docs/migrating-from-v1-to-v2.md]: I think this change is less clear than the previous version, as it excludes some information. What about something like this?

```suggestion
The node `internal` object isn't meant for adding node data. While Gatsby v1 allows this behaviour we now validate against it for v2. Node data should be added as fields on the top-level node object.

[Check the Node interface docs](https://www.gatsbyjs.org/docs/node-interface/) for allowed fields.
```

**Comment by endormi** [comment]
In my opinion they were fine as is, but those are definitely improvements.

File: docs/docs/migrating-from-v1-to-v2.md (hunk 1 above)
Comment (thatrobotdev) [reply docs/docs/migrating-from-v1-to-v2.md]: Thank you, this looks great!

File: docs/docs/migrating-from-v1-to-v2.md (hunk 2 above)
Comment (thatrobotdev) [reply docs/docs/migrating-from-v1-to-v2.md]: I was a little confused with this sentence. Thank you!

File: docs/docs/migrating-from-v1-to-v2.md (hunk 3 above)
Comment (thatrobotdev) [reply docs/docs/migrating-from-v1-to-v2.md]: I like your fix a lot better. Thanks!

**Pull request merged**